### PR TITLE
Admin doc reorganization + FAQs 

### PR DESCRIPTION
**Pull request by jasonblais**
There are two parts to this PR:

1. Reorganizing admin docs with this flow:
 - install Mattermost
 - deploy Mattermost --> need to know about onboarding, importing/migrating data, deploying Desktop App and mobile apps, options for encryption and image proxy, customization options, certificate-based auth
 - upgrading Mattermost --> upgrade/downgrade guides, important notes, changelog, version archive, ESR
 - administration --> config settings, CLI and other admin docs
 - mobile apps
 - user provisioning (auth)
 - access control (permissions)
 - scale
 - compliance

2. FAQs on the following topics (for a large prospect)
§  SSO Authentication [covered by existing docs]
§  Provisioning Workflow [new doc]

3. The following FAQs are incoming, but in separate PRs
§  Mobile Containerisation
§  Topology to support 50K users Day 1 and but scalable to 100K plus (single instance – and potential impact to performance)
§  On Prem vs Cloud Based – Considerations
§  Data Retention Requirements
§  Ethical Walls Capability
§  End to End Encryption (Cloud based Solution)

### CODE REVIEW

**Comment by jasonblais** [comment]
Good feedback on the TOC @wiersgallak! I'll see if I can incorporate some of these in this PR as well.

Thoughts on if 3 and 4 should be flipped on your list - would admin configure server before mobile?

**Comment by wiersgallak** [comment]
Switching 3 and 4 makes great sense.  Please let me know if I can help with the reorg in anyway. I found this helpful doc yesterday for moving pages to different folders (https://help.github.com/articles/moving-a-file-to-a-new-location/ - see number 3).

**Comment by jasonblais** [comment]
@wiersgallak Would you have time to help with the reorg? It would be really helpful actually :) 🙏 

Side note: I'm thinking we would keep the file locations the same, as changing their folder would also change the page URL, potentially leading to broken links elsewhere.

Otherwise, I updated the feedback on the user provisioning piece, appreciate it.

**Comment by wiersgallak** [comment]
@jasonblais For sure - more than happy to lend a hand.  This is something that I have been thinking about wanting to tackle in the near future anyways :) .   Do you want to do a separate PR for the reorg, or I could collaborate on this one? A new one may make it a bit cleaner and will not hold up the FAQs you have so far.

**Comment by wiersgallak** [comment]
@jasonblais I went ahead and made revisions since I had them fresh in my mind. If there is any section that you think needs to be rearranged or organized differently please let me know.

**Comment by jasonblais** [comment]
Looks great, thanks @wiersgallak!

File: source/guides/administrator.rst [hunk 1]
```diff
@@ -14,9 +16,11 @@ Getting Started
    /getting-started/welcome_email.rst
 
 
-Installing Mattermost
+Install Mattermost
```
Comment (amyblais) [inline source/guides/administrator.rst]: If we change these titles, then we'll need to update them elsewhere in the documentation if they are used as a reference in other docs.

**Pull request merged**